protocol/bin/enabled-calls.json

```diff
@@ -118,6 +118,7 @@
   "keybase.1.config.setProxyData": {"promise": true},
   "keybase.1.config.getProxyData": {"promise": true},
   "keybase.1.contacts.getContactsForUserRecommendations": {"promise": true},
+  "keybase.1.contacts.lookupContactList": {"promise": true},
```

Comment:
I think @zapu said we want to use `userSearch.userSearch` here

Reply:
Yup, working on migrating it over now. Didn't get to it on friday because I was working on the ssh ca stuff.